services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -79,19 +74,7 @@ testsPublic s =
           GET /team/{tid}/members - show legal hold status of all members
 
       -}
-      testGroup
-        "settings.legalholdEnabledTeams" -- FUTUREWORK: ungroup this level
-        [ testGroup -- FUTUREWORK: ungroup this level
-            "teams listed"
-            [ testGroup
-                "Users are invited to a group conversation."
-                [ testGroup
-                    "The group conversation contains legalhold activated users."
-                    [testOnlyIfLhWhitelisted s "If any user in the invite has not given consent then the invite fails" testNoConsentCannotBeInvited]
-                ],
-              test s "bench hack" testBenchHack
```

Comment:
you sure you want to remove this line?  it's a bit of a stupid test, but i feel it's nice to have it there and compile, and if we want we can enable it manually in the code.

if you remove it check if testBenchHack isn't dead code now.

Reply:
sorry, didn't see it's moved to /integration.  all good!